src/pip/_internal/vcs/subversion.py

```diff
@@ -33,6 +37,36 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def get_vcs_version(self):
+        # type: () -> Optional[Tuple[int, ...]]
+        """Return the version of the currently installed Subversion client.
+
+        :return: A tuple containing the parts of the version information or
+            ``None`` if the version returned from ``svn`` could not be parsed.
+        :raises: BadCommand: If ``svn`` is not installed.
+        """
+        version_prefix = 'svn, version '
+        version = self.run_command(['--version'], show_stdout=False)
+        if version.startswith(version_prefix):
+            version = version[len(version_prefix):].split()[0]
+        else:
+            return None
+
+        # Example versions:
+        #   svn, version 1.10.3 (r1842928)
+        #      compiled Feb 25 2019, 14:20:39 on x86_64-apple-darwin17.0.0
+        #   svn, version 1.7.14 (r1542130)
+        #      compiled Mar 28 2018, 08:49:13 on x86_64-pc-linux-gnu
```

Comment:
This comment would be better before the line where you strip out the version prefix I think, because these strings contain the version prefix, so the information is useful at that point, too.